sdk/storage/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+project (azure-storage LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+find_package(LibXml2 REQUIRED)
+
+option(BUILD_STORAGE_SAMPLES "Build sample codes" ON)
+
+set(AZURE_STORAGE_HEADER
+    inc/blobs/blob.hpp
+    inc/blobs/internal/protocol/blob_rest_client.hpp
+    inc/common/internal/protocol/rest_client_utility.hpp
+    inc/common/xml_wrapper.hpp
+)
+
+set(AZURE_STORAGE_SOURCE
+    src/blobs/internal/protocol/blob_rest_client.cpp
+    src/common/internal/protocol/rest_client_utility.cpp
+    src/common/xml_wrapper.cpp
+)
+
+add_library(azure-storage ${AZURE_STORAGE_HEADER} ${AZURE_STORAGE_SOURCE})
+
+target_include_directories (azure-storage PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc> $<INSTALL_INTERFACE:include/azure_storage>)
+
+target_link_libraries(azure-storage azure-core azure-transport-curl ${LIBXML2_LIBRARIES})
```

Comment:
In this PR, we haven't depended on libxml2 yet. So maybe `${LIBXML2_LIBRARIES}` should not be added in this PR.

Reply:
Also, if you added `${LIBXML2_LIBRARIES}`, you should also add `${LIBXML2_INCLUDE_DIR}`. You cannot just add one of them and leave out the other.

Reply:
I'll remove this.